### ISSUE TITLE
PGGAN blending issue

### ISSUE THREAD

**Issue by GongXinyuu** [facebookresearch/pytorch_GAN_zoo#98]
https://github.com/facebookresearch/pytorch_GAN_zoo/blob/382a96816f65235d2467b4d13ca4c4ed587f874a/models/networks/progressive_conv_net.py#L345

Should this line be `mergeLayer = self.alpha > 0 and len(self.fromRGBLayers) > 1`?

As the previous line calculate `y` on the condition of 
https://github.com/facebookresearch/pytorch_GAN_zoo/blob/382a96816f65235d2467b4d13ca4c4ed587f874a/models/networks/progressive_conv_net.py#L336

**Comment by Molugan** [comment]
You're right, the size of fromRGBLayers and scaleLayers are supposed to be the same at each step, but it's not super clean to refer to both in the code. I'll clean that up.

**Comment by LiUzHiAn** [comment]
Hi,

I don't think the size of fromRGBLayers and scaleLayers are the exactly same. Since the scale0 was not added to the scaleLayers.

https://github.com/facebookresearch/pytorch_GAN_zoo/blob/b75dee40918caabb4fe7ec561522717bf096a8cb/models/networks/progressive_conv_net.py#L264-L273

But the RGBLayer for the scale0 was added.
https://github.com/facebookresearch/pytorch_GAN_zoo/blob/b75dee40918caabb4fe7ec561522717bf096a8cb/models/networks/progressive_conv_net.py#L254-L257

So, `mergeLayer = self.alpha > 0 and len(self.fromRGBLayers) > 1` may be more reasonable. Did I misunderstand?

Thanks for your work.
